Makefile

```diff
@@ -286,3 +286,5 @@ uninstall:
 .SUFFIXES:
 
 -include $(wildcard *.d)
```

Comment:
This needs to be `build/*.d` now.  (And it should also include `tests/*.d`, that's a pre-existing bug.)